addons/web_editor/static/lib/odoo-editor/src/utils/sanitize.js

```diff
@@ -220,16 +222,40 @@ class Sanitize {
                 this._parse(firstChild);
             }
 
+            // Remove link ZWS not in selection
```

Comment:
Ah no, apologies, I just really kept naming them that way on purpose because for all intents and purposes they're the same. The particular characters are 200B and FEFF, which are variations on the concept of ZWS. I find it easier to just call them ZWS when it's the concept we're concerned with and not the specific character code.
But that's not a hill I'm willing to die on so if you want me to write ZWNBSP, I'll do that, I don't mind.